packages/core/src/main.ts

```diff
@@ -169,6 +169,10 @@ export async function createHoprNode(
   try {
     const dbPath = path.join(options.dataPath, 'db')
     await db.init(options.createDbIfNotExist, dbPath, options.forceCreateDB, options.environment.id)
+
+    // Dump entire database to a file if given by the env variable
+    const dump_file = process.env.DB_DUMP ?? ''
+    if (dump_file.length > 0) db.dumpDatabase(dump_file)
```

Comment:
```suggestion
    if (dump_file.length > 0) {
      db.dumpDatabase(dump_file)
    }
```
Please *always* use curly brackets. Otherwise, it's very inconvenient to analyze